examples/bind-shopping.proposed.amp.html

```diff
@@ -0,0 +1,271 @@
+      <amp-img src="./shirts/blue.jpeg" width=1 height=1 layout=responsive></amp-img>
+      <amp-img src="./shirts/dark-green.jpeg" width=1 height=1 layout=responsive></amp-img>
+      <amp-img src="./shirts/light-gray.jpeg" width=1 height=1 layout=responsive></amp-img>
+      <amp-img src="./shirts/white.jpeg" width=1 height=1 layout=responsive></amp-img>
+    </amp-carousel>
+
+    <amp-img class="color" width=355 height=50 src="https://placehold.it/355x50?text=COLOR%20???"
+        bind-src="https://placehold.it/300x400?text=COLOR%20{{items[selected].name}}">
+    </amp-img>
+
+    <div class="thumbnails">
+      <amp-img src="./shirts/black.jpeg" width=50 height=50 layout=fixed
+          on="tap:setState({selected: 0})"
+          bind-attr="{selected: selected === 0}"
+          bind-class="{outOfStock: !items[0].sizes.length}">
+      </amp-img>
+
+      <amp-img src="./shirts/brown.jpeg" width=50 height=50 layout=fixed
+          on="tap:setState({selected: 1})"
+          bind-attr="{selected: selected === 1}"
+          bind-class="{outOfStock: !items[1].sizes.length}">
+      </amp-img>
+
+      <amp-img src="./shirts/gray.jpeg" width=50 height=50 layout=fixed
+          on="tap:setState({selected: 2})"
+          bind-attr="{selected: selected === 2}"
+          bind-class="{outOfStock: !items[2].sizes.length}">
+      </amp-img>
+
+      <amp-img src="./shirts/navy.jpeg" width=50 height=50 layout=fixed
+        on="tap:setState({selected: 3})"
+        bind-attr="{selected: selected === 3}"
+        bind-class="{outOfStock: !items[3].sizes.length}">
+      </amp-img>
+
+      <amp-img src="./shirts/wine.jpeg" width=50 height=50 layout=fixed
+        on="tap:setState({selected: 4})"
+        bind-attr="{selected: selected === 4}"
+        bind-class="{outOfStock: !items[4].sizes.length}">
+      </amp-img>
+
+      <amp-img src="./shirts/blue.jpeg" width=50 height=50 layout=fixed
+        on="tap:setState({selected: 5})"
+        bind-attr="{selected: selected === 5}"
+        bind-class="{outOfStock: !items[5].sizes.length}">
+      </amp-img>
+
+      <amp-img src="./shirts/dark-green.jpeg" width=50 height=50 layout=fixed
+        on="tap:setState({selected: 6})"
+        bind-attr="{selected: selected === 6}"
+        bind-class="{outOfStock: !items[6].sizes.length}">
+      </amp-img>
+
+      <amp-img src="./shirts/light-gray.jpeg" width=50 height=50 layout=fixed
+        on="tap:setState({selected: 7})"
+        bind-attr="{selected: selected === 7}"
+        bind-class="{outOfStock: !items[7].sizes.length}">
+      </amp-img>
+
+      <amp-img src="./shirts/white.jpeg" width=50 height=50 layout=fixed
+        on="tap:setState({selected: 8})"
+        bind-attr="{selected: selected === 8}"
+        bind-class="{outOfStock: !items[8].sizes.length}">
+      </amp-img>
+    </div>
+
+    <amp-accordion>
+      <section expanded bind-attr="{expanded: !size}">
```

Comment:
> One approach is if expression evaluates to a pure boolean we toggle, otherwise we assign.

Yep, I suggested something doing similar to Angular's [ngClass](https://angular.io/docs/ts/latest/api/common/index/NgClass-directive.html) in the PR summary.